components/table/table-wrapper.js

```diff
@@ -62,6 +63,15 @@ export const tableStyles = css`
 	th.d2l-table-header-col-sortable {
 		padding: 0;
 	}
+	.d2l-checkbox,
+	d2l-input-checkbox,
+	d2l-selection-select-all,
+	input[type='checkbox'] {
+		margin-block: calc(-1 * ${cssSizes.inputBoxSize}rem); /* The height of an input checkbox */
```

Comment:
This would be setting the top and bottom margin to each be `-24px`. Is that what we want?
We can likely make a smarter equation here. We have the following information:
- Height we want (48px)
- Checkbox height (24px)
- Padding (0.75rem)
With this information we can calculate how much the top and bottom margins would need to change.